lib/IRGen/GenMeta.cpp

```diff
@@ -4041,7 +4052,8 @@ irgen::emitClassFragileInstanceSizeAndAlignMask(IRGenFunction &IGF,
   // FIXME: The below checks should capture this property already, but
   // resilient class metadata layout is not fully implemented yet.
   auto expansion = IGF.IGM.getResilienceExpansionForLayout(theClass);
-  if (IGF.IGM.isResilient(theClass, expansion)) {
+  if (IGF.IGM.isResilient(theClass, expansion)
+      || theClass->getParentModule() != IGF.IGM.getSwiftModule()) {
```

Comment:
The isResilient() check is redundant because a class is never resilient inside its own module